main/lsp/LSPTypechecker.cc

```diff
@@ -336,80 +446,68 @@ void LSPTypechecker::pushDiagnostics(TypecheckRun run) {
             continue;
         }
         ENFORCE(diagnosticEpochs[file.id()] <= epoch);
-        string uri;
-        { // uri
-            if (file.data(*gs).sourceType == core::File::Type::Payload) {
```

Comment:
This logic is now part of `fileRef2Uri`; I must have forgotten to update this particular code path.